package.json

```diff
@@ -93,11 +93,10 @@
     "lint:tests": "yarn lint packages/*/src/components/*/__tests__/*.test.*",
     "prepare": "ts-patch install -s",
     "serve:docs": "yarn workspace @patternfly/react-docs serve",
-    "serve:integration": "lerna run serve:demo-app",
+    "serve:integration": "lerna run preview --scope=demo-app-ts",
     "start": "yarn build && concurrently --kill-others \"yarn watch\" \"yarn workspace @patternfly/react-docs develop\"",
     "start:cypress": "lerna run cypress:open",
-    "start:demo-app": "lerna run start:demo-app --stream",
-    "start:demo-app:hot": "lerna run start:demo-app:hot --stream",
```

Comment:
Hot reloading is default in Vite during development, so it is no longer needed to keep this separate script around.